KeeTrayTOTP/Helpers/ImageExtensions.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Drawing;
+using KeePass.UI;
+
+namespace KeeTrayTOTP.Helpers
+{
+    internal static class ImageExtensions
+    {
+        public static Image CreateImageFromColor(Color color)
```

Comment:
I had noticed this in the screenshot. Until today I did not know you can set a colour in the KeePass database settings.